hardhat.config.ts

```diff
@@ -12,7 +12,13 @@ export default {
   solidity: {
     compilers: [
       {
-        version: '0.6.12',
+        version: '0.7.6',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
```

Comment:
Consider increasing the number of runs to 9000 and seeing if it improves gas efficiency.

Reply:
Increasing runs to `9000`, doesnt improve noticeably

<pre>
·--------------------------------------------------|---------------------------|--------------|----------------------------·
|               Solc version: 0.7.6                ·  Optimizer enabled: true  ·  Runs: 9000  ·  Block limit: 9500000 gas  │
···················································|···························|··············|·····························
|  Methods                                                                                                                 │
·····················|·····························|·············|·············|··············|··············|··············
|  Contract          ·  Method                     ·  Min        ·  Max        ·  Avg         ·  # calls     ·  usd (avg)  │
·····················|·····························|·············|·············|··············|··············|··············
|  Orchestrator      ·  addTransaction             ·      71318  ·     131876  ·      109945  ·           6  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  Orchestrator      ·  rebase                     ·      27292  ·     123293  ·       80519  ·          80  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  Orchestrator      ·  removeTransaction          ·      28441  ·      51244  ·       37462  ·           3  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  Orchestrator      ·  setTransactionEnabled      ·      25783  ·      29971  ·       27877  ·           2  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  Orchestrator      ·  transferOwnership          ·          -  ·          -  ·       30781  ·           1  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  approve                    ·      16674  ·      46746  ·       36628  ·          34  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  decreaseAllowance          ·      28507  ·      32787  ·       30647  ·           8  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  increaseAllowance          ·      32795  ·      47819  ·       38807  ·          10  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  permit                     ·      76621  ·      76633  ·       76629  ·           3  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  rebase                     ·      27579  ·      38298  ·       35084  ·          13  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  setMonetaryPolicy          ·          -  ·          -  ·       47254  ·           8  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  transfer                   ·      39769  ·      54745  ·       53379  ·          11  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  transferAll                ·      31058  ·      39458  ·       37358  ·           4  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  transferAllFrom            ·      31016  ·      46016  ·       34494  ·           5  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragments        ·  transferFrom               ·          -  ·          -  ·       46440  ·           2  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setCpiOracle               ·      31162  ·      46162  ·       38662  ·           4  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setDeviationThreshold      ·      15099  ·      30159  ·       18111  ·           5  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setMarketOracle            ·      31161  ·      46161  ·       38661  ·           4  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setOrchestrator            ·      31138  ·      46126  ·       38632  ·           4  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setRebaseLag               ·          -  ·          -  ·       30134  ·           2  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  UFragmentsPolicy  ·  setRebaseTimingParameters  ·      25435  ·      40471  ·       29379  ·          15  ·          -  │
·····················|·····························|·············|·············|··············|··············|··············
|  Deployments                                     ·                                          ·  % of limit  ·             │
···················································|·············|·············|··············|··············|··············
|  Orchestrator                                    ·          -  ·          -  ·      986760  ·      10.4 %  ·          -  │
···················································|·············|·············|··············|··············|··············
|  UFragments                                      ·          -  ·          -  ·     1614953  ·        17 %  ·          -  │
···················································|·············|·············|··············|··············|··············
|  UFragmentsPolicy                                ·          -  ·          -  ·     1166050  ·      12.3 %  ·          -  │
·--------------------------------------------------|-------------|-------------|--------------|--------------|-------------·
</pre>